src/librustc_target/spec/powerpc_wrs_vxworks_spe.rs

```diff
@@ -20,7 +20,6 @@ pub fn target() -> TargetResult {
         options: TargetOptions {
             // feature msync would disable instruction 'fsync' which is not supported by fsl_p1p2
             features: "+secure-plt,+msync".to_string(),
-            target_mcount: "_mcount".to_string(),
```

Comment:
"mcount" (or the name we specify here) is the function called by compiler generated instrumentation code when profiling is enabled. 

https://www.kernel.org/doc/Documentation/trace/ftrace-design.txt
https://ftp.gnu.org/pub/old-gnu/Manuals/gprof-2.9.1/html_node/gprof_25.html

VxWorks would need to implement this function (we don't currently have it). Removing this code doesn't actually disable calls to mcount, it just reverts to the default name "mcount". 

For clarity I suggest adding a line to vxworks_base.rs:

`target_mcount: "_mcount".to_string(), // VxWorks needs to implement this to support profiling`

Reply:
Done.